src/subscribe/index.html

```diff
@@ -73,6 +73,10 @@
                 </section>
                 <section class="form-row">
                     <h5>Organization</h5>
+                    <p>
+                        You will start a sixty-day free trial when you complete signup. We collect your credit card
+                        now so that you can seamlessly continue using Stockpile when your trial runs out.
+                    </p>
```

Comment:
Is collecting their credit card for the free trial required by Stripe? It might be nice to leave it optional. I know I never like to put my credit card for free trials and I like the idea of making the time to subscribe as short as possible.

If it is required, it might be good to add something along the line of "You can cancel your subscription anytime and we will notify you before your free trial ends".